spec/services/external_payment_processor_spec.rb

```diff
@@ -33,4 +33,46 @@
     end
     
   end
+  
+  describe :create_customer_subscription do
+    
+    context "with valid card details" do
+      it "creates a customer", :vcr do
```

Comment:
`:vcr` tag can be on the context above if all the examples within are using it :)